sdk/logs/src/main/java/io/opentelemetry/sdk/logs/internal/SdkEventBuilder.java

```diff
@@ -5,35 +5,83 @@
 
 package io.opentelemetry.sdk.logs.internal;
 
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.incubator.events.EventBuilder;
+import io.opentelemetry.api.incubator.logs.AnyValue;
+import io.opentelemetry.api.incubator.logs.ExtendedLogRecordBuilder;
 import io.opentelemetry.api.logs.LogRecordBuilder;
+import io.opentelemetry.api.logs.Severity;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.common.Clock;
 import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
 class SdkEventBuilder implements EventBuilder {
+
+  private static final AttributeKey<String> EVENT_NAME = AttributeKey.stringKey("event.name");
```

Comment:
Can we use the `SemanticAttributes.EVENT_NAME` here instead?

Reply:
maybe once it's stable, but I think we've discussed not relying on unstable modules from the SDK

Reply:
Correct. We copy in a variety of other semantic convention attributes to avoid version compatibility issues, including [service.name](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ResourceConfiguration.java#L35), [telemetry.sdk.*](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java#L30-L35), [and several others for the zipkin exporter](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/OtelToZipkinSpanTransformer.java#L36-L49). I think its our only choice until `semantic-conventions-java` reaches a more mature point. And even then, it might still make sense.

Reply:
> maybe once it's stable, but I think we've discussed not relying on unstable modules from the SDK

Even in an unstable portion of the sdk? That seems a little heavy handed. I won't lose sleep over it tho.